arviz/plots/backends/bokeh/bokeh_parallelplot.py

```diff
@@ -3,26 +3,18 @@
 from bokeh.models.tickers import FixedTicker
 import numpy as np
 
+from ....rcparams import rcParams
+
 
 def _plot_parallel(ax, diverging_mask, _posterior, var_names, figsize, show):
     if ax is None:
-        tools = ",".join(
-            [
-                "pan",
-                "wheel_zoom",
-                "box_zoom",
-                "lasso_select",
-                "poly_select",
-                "undo",
-                "redo",
-                "reset",
-                "save,hover",
-            ]
-        )
+        tools = rcParams["plot.bokeh.tools"]
+        output_backend = rcParams["plot.bokeh.output_backend"]
+        dpi = rcParams["plot.bokeh.figure.dpi"]
```

Comment:
This pattern means if people want to change things they need to both modify RcParams or arg, so api is sort of split. Is my understanding correct?

Reply:
Api is split now, until we fix the api

Reply:
Works for me